arch/arm/boot/dts/qcom/qcom-msm8974-sony-xperia-rhine-togari.dts

```diff
@@ -1,6 +1,9 @@
 // SPDX-License-Identifier: GPL-2.0
 #include "qcom-msm8974-sony-xperia-rhine.dtsi"
 
+/* This device has Maxim touchscreen instead of Synaptics touchscreen. */
+/delete-node/ &blsp1_i2c2;
```

Comment:
Don't delete the i2c node that doesn't make any sense. 

There's several options. 

1. If your device has the maxim touchscreen on the same i2c bus then you can delete that original touchscreen node
2. Or you can do &touchscreen { status = "disabled"; }
3. Or you can move the touchscreen node from common.dtsi to the devices that do have synaptics touchscreen, as the touchscreen is obviously not common across all Rhine devices. And then you don't need to do anything in this dts file you're writing, just in *Rhine*.dts*

Reply:
hi, i have added an initial config for maxim touchscreen and disable the touch screen node from rhibe common.